model/github/github_repository.rb

```diff
@@ -54,14 +57,18 @@ def after_destroy
     destroy_blob_storage if access_key
   end
 
+  def refresh_blob_storage_token
```

Comment:
For the existing tokens, it looks ok to refresh them as well since we add session_token to them. Still, can you please make sure it is ok?